### PR TITLE
Proposed format and initial links

### PR DESCRIPTION
**Pull request by numerative**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+A wiki :ledger: for Udacity Google India Challenge Scholarship 2018 Android Basics Track.
+<p align="center">
+  <img width="300" height="300" src="https://imgur.com/FrGft2b.png">
+</p>
+
+### General
+* [Archived Slack Messages - Slark](https://slarck.com/9z1N5S/?k=Xkqwg9YczIT9xRPqAnMvPj7brNC3F8qFw5dOyS8M)
+* [Forums](https://discussions.udacity.com/c/standalone-courses/ud834-in/)
+* New Workspace Link - _to be updated after May 29th_
+* [LinkedIn Profiles](https://docs.google.com/spreadsheets/d/1JqGTMa6AQJTjHZT1w7tU76-9h6K3vdxAj1A5yNlk3mg) _Not recommended as public documents can be abused outsiders_
```
Comment (piedcipher) [inline README.md:12]: Will it be fine if we keep this (line 12) in README?

Comment (numerative) [reply README.md:12]: I'm only concerned about some outsider changing it and I don't know a solution. What should we do?

Comment (piedcipher) [reply README.md:12]: I guess I misread that line, My bad.

**Review by piedcipher — APPROVED** [review]

**Comment by piedcipher** [comment]
Great Job @numerative! :clap: Thank you very much for contributing :+1: 
Merged #1 :tada:

File: README.md (hunk 1 above)
Comment (piedcipher) [reply README.md:12]: We can try talking about this with owner of spreadsheet.

Comment (numerative) [reply README.md:12]: Sure. Will do.

**Pull request merged**